servers.json

```diff
@@ -30,6 +30,13 @@
             "menastro.gg"
         ]
     },
+    {
+        "id": "fusionmc",
+        "name": "FusionMC Network",
+        "addresses": [
+            "play.fusionmc.ml"
```

Comment:
Please remove the subdomain part from this address, and you must include your server logo alongside these changes.